OpenRA.Game/Actor.cs

```diff
@@ -320,6 +322,30 @@ public bool CanBeViewedByPlayer(Player player)
 			return defaultVisibility.IsVisible(this, player);
 		}
 
+		public IEnumerable<string> GetAllTargetTypes()
```

Comment:
I'm thinking these should go into some Actor exts, or (since targeting code is also in the engine, not in mod code) get their own region, because this class is getting pretty crowded.